src/main/java/org/galatea/starter/Application.java

```diff
@@ -40,4 +40,4 @@ public void run(final ApplicationArguments args) {
       throw new MissingOptionException("Server port must be set via command line parameter");
     }
   }
-}
+}
```

Comment:
This change seems unnecessary

Reply:
How do I undo this change?

Reply:
This is the type of change that's easy to undo _before_ commit/pushing, but now that you've pushed it's a little harder to undo just it. Best action would be to just change the file back to what it was and commit that (with all the other PR changes)